src/main/java/hudson/remoting/Launcher.java

```diff
@@ -787,6 +790,10 @@ public static boolean isWindows() {
         return File.pathSeparatorChar==';';
     }
 
+    public static String getCommunicationProtocol() {
```

Comment:
Could you please add Javadoc to this method and to `getProtocolName()`? They are very close, and disambiguation is needed. Also, it could be a "getCommunicationProtocolName"

Reply:
I understand. Thank you for reviewing!